assert/not-installed.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Assert databricks CLI is not already installed
+if which databricks >/dev/null 2>&1; then
+    exit 1
+else
+    exit 0
+fi
```

Comment:
FWIW, this shortens to `! which databricks`.

Reply:
Yeah. I personally find the current approach clearer and more explicit and consistent with the other assertion definitions though.